youtube_dl/extractor/generic.py

```diff
@@ -2978,6 +2988,14 @@ def merge_dicts(dict1, dict2):
                     merged[k] = v
             return merged
 
+        # Look for Share-Videos.se embeds
+        sharevideosse_urls = [m.group('url') for m in re.finditer(
+            r'<iframe[^>]+?src\s*=\s*(["\'])(?P<url>https?://embed\.share-videos\.se/auto/embed/\d+\?.+)\1',
```

Comment:
Incorrect. Should not be greedy.

Reply:
Do you have any suggestion for that?

Reply:
Suggestion for what? Making regex non greedy? Maybe you'll finally read some doc on regexes?